components/plugin_marketplace/marketplace_modal.scss

```diff
@@ -48,6 +48,17 @@
         .more-modal__row {
             align-items: normal;
 
+            .tag {
+                padding: 3px 5px;
+                margin-left: 10px;
+                background-color: $gray;
+                border-radius: 4px;
+                font-weight: bold;
+                font-size: 0.8em;
+                color: $white;
+                cursor: help;
```

Comment:
During the demo I noticed that the cursor change when over the tag made it hard to read the label text. Also, not sure if I've seen this type of cursor any other places in the webApp for tooltips.

Thoughts @asaadmahmood ?